src/adapters/gam/managers/creatives.py

```diff
@@ -420,29 +478,40 @@ def _create_hosted_asset_creative(self, asset: dict[str, Any]) -> dict[str, Any]
         """Create a hosted asset (image/video) creative for GAM."""
         width, height = self._get_creative_dimensions(asset)
 
-        # Upload the binary asset to GAM
-        uploaded_asset = self._upload_binary_asset(asset)
-        if not uploaded_asset:
-            raise Exception("Failed to upload binary asset")
+        # Get the creative URL
+        url = asset.get("url")
+        if not url:
+            raise Exception("No URL found for hosted asset creative")
 
         # Determine asset type
         asset_type = self._determine_asset_type(asset)
 
+        # Get click-through URL (required by GAM for redirect creatives)
+        # Use clickthrough_url from asset, landing_url, or fall back to image URL itself
+        # GAM rejects example.com and other placeholder domains
+        destination_url = asset.get("clickthrough_url") or asset.get("landing_url") or url
+
+        # For now, use ImageRedirectCreative which references external URL
+        # This avoids needing to download and upload binary assets
```

Comment:
Then we should not accept image creative format for the GAM adapter. Maybe break out a new format that only accepts image URLs.